### PR TITLE
Updated cookiecutter templates to use GitHub actions for CI

### PR DESCRIPTION
**Pull request by mraarif**
**JIRA:**
https://openedx.atlassian.net/browse/BOM-2131

### CODE REVIEW

**Review by mzulqarnain1 — APPROVED** [review]

File: cookiecutter-django-app/{{cookiecutter.repo_name}}/.github/workflows/pypi-publish.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Publish package to PyPi
```
Comment (jinder1s) [inline cookiecutter-django-app/{{cookiecutter.repo_name}}/.github/workflows/pypi-publish.yml:1]: Were you not able to get this from the python-template? This seems to be an exact copy from python-template

Comment (mraarif) [reply cookiecutter-django-app/{{cookiecutter.repo_name}}/.github/workflows/pypi-publish.yml]: I thought python-template was only used as a base for python-library, you're right we don't this here.

File: cookiecutter-django-app/{{cookiecutter.repo_name}}/.github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+name: Python CI
```
Comment (jinder1s) [inline cookiecutter-django-app/{{cookiecutter.repo_name}}/.github/workflows/ci.yml:1]: This file is also a duplicate

Comment (jinder1s) [reply cookiecutter-django-app/{{cookiecutter.repo_name}}/.github/workflows/ci.yml:1]: This should also be layered in from the python-template

Comment (mraarif) [reply cookiecutter-django-app/{{cookiecutter.repo_name}}/.github/workflows/ci.yml:1]: workflow setup in django-app is setting up specific Django versions for CI that python-library doesn't need

Comment (jinder1s) [reply cookiecutter-django-app/{{cookiecutter.repo_name}}/.github/workflows/ci.yml:1]: ah okay, sorry, I should have looked more carefully.

**Review by jinder1s — APPROVED** [review]

**Pull request merged**